app/views/foods/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="is-flex is-justify-content-end">
+   <%= link_to "Add Food", new_food_path, class: "button is-focused is-black m-5"%>
+</div>
+<div class="table-container m-3">
+  <table class="table is-fullwidth is-striped is-bordered">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Measurement Unit</th>
+        <th>Unit Price</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @foods.each do |food| %>
+        <tr>
+          <th><%= food.name %></th>
+          <td><%= food.measurement_unit %></td>
+          <td>$<%= food.price %></td>
+          <td><%= link_to 'Delete', food, method: :delete, data: { confirm: 'Are you sure you want to delete this food?'}, class: "button is-small is-danger is-outlined"%></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</div>
```

Comment:
- [ ] `Can delete a food item from the lists - only by owners`, however, when I try to login in as another user I have access to the other's items so that I can destroy their list or modify. please make sure you can't modify other's items once you are not an owner.

Reply:
Fixed